### PR TITLE
Bridges inegration testing script nit - externalize `polakdot-sdk` path

### PR DESCRIPTION
**Pull request by bkontur**
This PR contains a minor update to the bridges testing scripts, allowing us to externalize the `polkadot-sdk` path instead of downloading it + updated sudo patch file.

- [X] Does not require a CHANGELOG entry

### CODE REVIEW

File: integration-tests/bridges/sudo-relay.patch [hunk 1]
```diff
@@ -1,74 +1,209 @@
...
 +	"pallet-sudo/std",
  	"pallet-timestamp/std",
  	"pallet-transaction-payment-rpc-runtime-api/std",
  	"pallet-transaction-payment/std",
+diff --git a/relay/kusama/src/genesis_config_presets.rs b/relay/kusama/src/genesis_config_presets.rs
+index dda349608..4714ecd51 100644
+--- a/relay/kusama/src/genesis_config_presets.rs
++++ b/relay/kusama/src/genesis_config_presets.rs
+@@ -188,6 +188,9 @@ fn kusama_testnet_genesis(
+			"forceEra": Forcing::NotForcing,
```
Comment (pandres95) [inline integration-tests/bridges/sudo-relay.patch:192]: Not exactly sure if this is the place. After applying the patch (using `git apply integration-tests/bridges/sudo-relay.patch` got the following error:

```
error: corrupt patch at line 192

```

File: integration-tests/bridges/run-test.sh [hunk 2]
```diff
@@ -7,21 +7,29 @@ trap 'kill -9 -$$ || echo "Environment already teared down"' SIGINT SIGTERM EXIT
...
-git fetch --tags
-git checkout polkadot-v1.11.0
-popd
+
+if [ -z "$FRAMEWORK_REPO_PATH" ]; then
+  # Download the bridge testing "framework" from the `polkadot-sdk` repo
+  # to `~/local_bridge_testing/downloads/polkadot-sdk`.
+  export DOWNLOADS_PATH=$LOCAL_BRIDGE_TESTING_PATH/downloads
+  echo "FRAMEWORK_REPO_PATH is NOT set, so downloading 'polkadot-sdk' repo to the: $DOWNLOADS_PATH"
+  exit 1
```
Comment (pandres95) [inline integration-tests/bridges/run-test.sh:16]: Does it make sense to `exit` immediately after informing the user the PSDK will be downloaded, or is this just a breakpoint that was forgotten to be removed?

**Review by pandres95 — CHANGES_REQUESTED** [review]
In general, looks okay, but had a couple minor issues trying to run it. Especially, one related with `run-test.sh` when `FRAMEWORK_REPO_PATH` is not set.

**Review by ggwpez — APPROVED** [review]

File: integration-tests/bridges/run-test.sh (hunk 2 above)
Comment (bkontur) [inline integration-tests/bridges/run-test.sh:16]:
```suggestion
```

Comment (bkontur) [reply integration-tests/bridges/run-test.sh]: :see_no_evil: uff, thank you, yes breakpoint

File: integration-tests/bridges/sudo-relay.patch (hunk 1 above)
Comment (bkontur) [reply integration-tests/bridges/sudo-relay.patch]: hmm, I didn't realize that. It looks like the modified patch file itself was also included in the patch file when I ran `git diff` :D
fixed: https://github.com/polkadot-fellows/runtimes/pull/435/commits/fdc0ef0039afff8f53863965676050676c73b523

Comment (bkontur) [reply integration-tests/bridges/sudo-relay.patch]: @pandres95 thank you :)

**Review by pandres95 — APPROVED** [review]

**Review by wirednkod — APPROVED** [review]

**Pull request merged**